c_common/models/live_packet_gather/src/live_packet_gather.c

```diff
@@ -113,52 +114,54 @@ static struct lpg_config config;
 
 //! \brief Because WHY OH WHY would you use aligned memory? At least with this
 //! we don't get data aborts.
-static inline void write_unaligned(
-        void *base, uint32_t index, uint32_t value) {
+static inline void write_word(void *base, uint32_t index, uint32_t value) {
     uint16_t *ary = base;
     uint32_t idx = index * 2;
     ary[idx++] = CLAMP16(value);
     ary[idx] = CLAMP16(value >> 16);
 }
 
+//! \brief Simple mirror of write_word() for true 16 bit values.
+static inline void write_short(void *base, uint32_t index, uint32_t value) {
```

Comment:
Not an important function, except that it makes the code dealing with 16-bit values look like the code dealing with 32-bit values, which makes reading what is going on _much_ easier.

Reply:
yep